packages/@o3r/core/schematics/ng-add/schema.json

```diff
@@ -1,9 +1,26 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ngAddSchematicsSchema",
-  "title": "Add Otter library",
-  "description": "ngAdd Otter library",
+  "title": "Add Otter Framework to an Angular project",
+  "description": "ngAdd Otter Framework to the Angular project",
   "properties": {
+    "preset": {
+      "description": "Preset of module list to automatically install",
+      "type": "string",
+      "default": "basic",
+      "enum": [
+        "basic",
+        "cms"
+      ],
+      "x-prompt": {
+        "message": "Which preset to use to start your application?",
+        "type": "list",
+        "items": [
+          { "value": "basic", "label": "BASIC - Minimum plugin list to install for a basic Otter application (no additional modules)" },
+          { "value": "cms", "label": "CMS - Add the Otter modules allowing the full administration of the application via the CMS" }
```

Comment:
should we link a MD from our readme for further details about what the CMS stack contains?

Reply:
I will do it but not sure the links in select are working in all consoles